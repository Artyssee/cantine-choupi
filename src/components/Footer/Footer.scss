@import '../../styles/variables/colors';
@import '../../styles/mixins/fonts';

.footerContainer {
	display: flex;
	justify-content: center;
	width: 100%;
	background-color: $color-black;
	margin: 0;
	padding: 0;
}

.footerContainerWrapper {
	display: flex;
	flex-direction: column;
	align-items: flex-start;
	width: 1200px;
	text-transform: uppercase;

	&Heading {
		@include LatoFont(24px, $color-grey-white, normal);
		margin-block-end: 20px;
		margin-block-start: 20px;

		text-decoration: none;
	}

	&Item {
		padding: 5px 0;
	}

	&Link {
		@include LatoFont(24px, $color-grey-white, normal);

		text-decoration: none;

		&:hover {
			cursor: pointer;
			border-bottom: 2px solid $color-grey-white;
			transition: .08s;
		}
	}
}
