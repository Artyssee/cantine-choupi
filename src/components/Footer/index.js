import React from 'react';
import Link from 'next/link';
import Styles from './Footer.scss';

const Footer = () => {
  return (
    <div className={Styles.footerContainer}>
      <ul className={Styles.footerContainerWrapper}>
        <h5 className={Styles.footerContainerWrapperHeading}>Pages</h5>
        <li className={Styles.footerContainerWrapperItem}>
          <Link href='/'>
            <a className={Styles.footerContainerWrapperLink}>Home</a>
          </Link>
        </li>
        <li className={Styles.footerContainerWrapperItem}>
          <Link href='/dishes'>
            <a className={Styles.footerContainerWrapperLink}>Food & Drinks</a>
          </Link>
        </li>
      </ul>
    </div>
  );
};

export default Footer;
