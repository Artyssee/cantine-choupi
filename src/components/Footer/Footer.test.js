import React from 'react';
import { shallow } from 'enzyme';
import Footer from './';
import Link from 'next/link';

const wrapperShallow = shallow(<Footer />);

describe('The Footer component', () => {
  it('renders correctly', () => {
    expect(wrapperShallow).toMatchSnapshot();
  });

  it('renders at least one link', () => {
    expect(wrapperShallow.find('ul').contains([
      <li key={1}>
        <Link href='/'>
          <a>Home</a>
        </Link>
      </li>
    ])).toBe(true);
  });
});
