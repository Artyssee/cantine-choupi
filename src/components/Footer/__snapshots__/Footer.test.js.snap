// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer component renders correctly 1`] = `ShallowWrapper {}`;

exports[`The Navigation component renders correctly 1`] = `ShallowWrapper {}`;
